sources/assets/bloodhound-ce/bloodhound-ce

```diff
@@ -10,5 +10,8 @@ if ! pg_isready -q; then
     service postgresql start
 fi
 
-/opt/tools/BloodHound-CE/bloodhound -configfile /opt/tools/BloodHound-CE/bloodhound.config.json
+if neo4j status|grep -q "not running"; then
+    neo4j start
```

Comment:
We need to wait for neo4j to start before running `bloodhound-ce` otherwise we get `Failed starting the server: failed to start services: graph migration error: ConnectivityError`.